rellicConfig.cmake.in

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2020-present, Trail of Bits, Inc.
+# All rights reserved.
+#
+# This source code is licensed in accordance with the terms specified in
+# the LICENSE file found in the root directory of this source tree.
+#
+
+@PACKAGE_INIT@
+
+include("${CMAKE_CURRENT_LIST_DIR}/@PROJECT_NAME@Targets.cmake")
+check_required_components("@PROJECT_NAME@")
```

Comment:
This file needs to include `find_dependency` calls to LLVM, Clang, glog, gflags, and all of the other public third party interfaces in the rellic library(s).

https://cmake.org/cmake/help/latest/module/CMakeFindDependencyMacro.html

Reply:
Addressed in latest commit.